README.md

```diff
@@ -105,6 +105,10 @@ avalanche-network-runner server \
 
 Note that the above command will run until you stop it with `CTRL + C`. You should run further commands in a separate terminal.
 
+_Note: If you set the environment variables `AVALANCHEGO_EXEC_PATH` and `AVALANCHEGO_PLUGIN_PATH` previously to launching the server,
```

Comment:
If the environment variables ... and ... are set, they will be used to start the server if no command line flags are passed.